csFastFloat/FastFloatParser.cs

```diff
@@ -676,7 +659,7 @@ unsafe static internal ParsedNumberString ParseNumberString(byte* p, byte* pend,
         {
           return answer;
         }
-        if (!Utils.is_integer(*p, out uint digit) && (*p != decimal_separator)) // culture info ?
+        if (!Utils.is_integer(*p, out uint _) && (*p != decimal_separator)) // culture info ?
```

Comment:
This is pure stylistic, to denote that the arugment isn't needed.
Emitted machine code will be the same.